docs/rule/require-input-label.md

```diff
@@ -2,11 +2,11 @@
 
 Users with assistive technology need `<input>` elements to have associated labels. It's so essential for users that failure to provide an associated label for an `<input>` element will cause failure of **three** different WCAG success criteria: [1.3.1, Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html), [3.3.2, Labels or Instructions](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html), and [4.1.2, Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html). It is also associated with common failure [#68: Failure of Success Criterion 4.1.2 due to a user interface control not having a programmatically determined name](https://www.w3.org/WAI/WCAG21/Techniques/failures/F68).
 
-This rule checks to see if the `<input>` element is contained by a label element. If it is not, it checks to see if the `<input>` element has any of these three attributes: `id`, `aria-label`, or `aria-labelledby`. While the `id` element on the `<input>` is not a concrete indicator of the presence of an associated `<label>` element with a `for` attribute, it is a good indicator that one likely exists.
+This rule checks to see if the input element (either `<input>`, `<textarea>`, or `<select>`) element is contained by a label element. If it is not, it checks to see if the input has any of these three attributes: `id`, `aria-label`, or `aria-labelledby`. While the `id` element on the input is not a concrete indicator of the presence of an associated `<label>` element with a `for` attribute, it is a good indicator that one likely exists.
```

Comment:
Can you move the list of elements to an actual list below this paragraph? It's much easier to glance at a list and it will be easier for us to update in the future too.

```
* `<input>`
* `<textarea>`
* `<select>`
```